org.eclipse.xtext/src/org/eclipse/xtext/build/Indexer.java

```diff
@@ -244,6 +244,17 @@ public URI getURI() {
 		}
 	}
 
+	/**
+	 * Return a new {@link ResolvedResourceDescription} from a {@link IResourceDescription}.
+	 *
+	 * @param description
+	 *            the resource description, must not be {@code null}
+	 * @return the new resolved resource description, never {@code null}
+	 */
+	protected Indexer.ResolvedResourceDescription getResolvedResourceDescription(IResourceDescription description) {
```

Comment:
misses `@since 2.26`

Reply:
Is it possible to make the API slightly more generic, e.g. return IResourceDescription instead of an implementation class. Something like `protected IResourceDescription toIndexingPhaseDescription(IResourceDescription description) {}` ?

Reply:
done